### DISCUSSION TITLE
Welcome to Planning-Poker Discussions!

### DISCUSSION THREAD

**Discussion by Carduelis** [Carduelis/poker-widget-preact#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eteare** [comment]
Hello! I am a product manager who is looking to use your planning poker app in Miro. I installed it and it's sitting in the app dropdown, but when I click on your logo nothing happens. Do you have any instructions on how to use the app? I am sure I am doing something wrong that is pretty simple. Any help appreciated!

**Reply by Carduelis** [reply]
Thank you for the feedback, we're getting there soon!
